pagopa/io-app
# feat: [IOBP-453] Addition of barcode regeneration screen

## Short description
addition of said screen
<img src="https://github.com/pagopa/io-app/assets/60693085/66cabb22-f6d0-466a-ad71-26c1081ce17d" width=200/>

## List of changes proposed in this pull request
- addition of screen
- small refactors 
- addition of required i18n keys

## How to test
with `io-dev-server` running, generate an idpay barcode (discount initiative->auth a payment->barcode) and wait for it to expire, then generate a new one and make sure that nothing fails and the app actually generated a new barcode.
